docs/design/og-image.card.html

```
@@ -0,0 +1,78 @@
+    .wordmark {
+      font-weight: 700; letter-spacing: 0.02em; line-height: 0.94;
+      color: var(--ink); font-size: 92px;
+    }
+    .wordmark .express { color: var(--a1); }
+    .tagline {
+      font-family: var(--mono); font-size: 35px;
+      color: color-mix(in srgb, var(--dim) 22%, var(--ink));
+      margin-top: 22px;
+    }
+  </style>
+</head>
```

HTMLHint (1.9.2)

[error] <title> must be present in <head> tag.

(title-require)